apps/scan/frontend/src/utils/use_voter_settings_controls.ts

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  VoterSettingsManagerContext,
+  useCurrentLanguage,
+  useLanguageControls,
+  useQueryChangeListener,
+} from '@votingworks/ui';
+import { DefaultTheme, ThemeContext } from 'styled-components';
+import { LanguageCode } from '@votingworks/types';
+import { getAuthStatus, getScannerStatus } from '../api';
+
+interface VoterSettingsControls {
+  resetVoterSettings: () => void;
+}
+
+/**
+ * useVoterSettingsControls adds Scan-specific settings to the base settings provided by VoterSettingsManagerContext
```

Comment:
really it aggregates three different contexts, will change comment

Reply:
I would like to see both comments changed to describe what this piece of code actually does, which IMO is:
- saving and reapplying a voter's theme and language settings when their voting session is interrupted by an election official
- resetting the settings after a voter finishes